examples/message/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "message",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "data-provider": "file:../../",
```

Comment:
I'm not entirely sure whether this is the correct way of setting local dependencies.

Reply:
Hmm strange, it works for me with both npm (5.5.1) and yarn (1.3.2) - node 8.9.1
@jurom had a similar problem and had to switch to node 8 (from node ~3?) using `nvm`...